nginx-controller/main.go

```diff
@@ -130,10 +130,32 @@ func main() {
 		}
 	}
 
+	cfg := nginx.NewDefaultConfig()
+	if *nginxConfigMaps != "" {
+		ns, name, err := controller.ParseNamespaceName(*nginxConfigMaps)
+		if err != nil {
+			glog.Fatalf("Error parsing the nginx-configmaps argument: %v", err)
+		}
+		cfm, err := kubeClient.CoreV1().ConfigMaps(ns).Get(name, meta_v1.GetOptions{})
+		if err != nil {
+			glog.Fatalf("Error when getting %v: %v", *nginxConfigMaps, err)
+		}
+		cfg = nginx.ParseConfigMap(cfm, *nginxPlus)
+		if cfg.MainServerSSLDHParamFileContent != nil {
+			fileName, err := ngxc.AddOrUpdateDHParam(*cfg.MainServerSSLDHParamFileContent)
+			if err != nil {
+				glog.Errorf("Configmap %s/%s: Could not update dhparams: %v", ns, name, err)
```

Comment:
let's exit the Ingress controller in case it fails to write the DHParam file -- use glog.Fatalf